web/src/test/auto/integrated/cases/element_interfaces.js

```diff
@@ -1182,6 +1220,10 @@ describe('Element Input/Output Interfacing', function() {
         });
       });
 
+      it.skip('getSelectedText', function() { // Not yet properly supported.
```

Comment:
Design-mode iframes and content editables are much trickier in regard to selected-text retrieval, since the text content could span multiple nodes, etc.  I opted for a '16.0-equivalent behavior' implementation instead for these element types, but that'd fail the test here as written.